src/composables/useEthers.ts

```diff
@@ -1,42 +1,52 @@
-import { APP } from '@/constants/app';
+import { ref } from 'vue';
 import {
   TransactionReceipt,
   TransactionResponse
 } from '@ethersproject/providers';
+
 import useAccountBalances from './useAccountBalances';
+import useBlocknative from './useBlocknative';
 import useTransactions from './useTransactions';
 
 type TxCallback = (
   txData: TransactionResponse,
   receipt?: TransactionReceipt
 ) => void;
 
+// keep a record of processed txs
+export const processedTxs = ref<Set<string>>(new Set(''));
```

Comment:
This was added so that there is no duplicate checking of transactions by the `wait` function with `getTransactionReciept`.

Essentially we are going to use `getTransactionReciept` if users some pending transactions after they refresh (after persisting to local storage)